hztb-servicemanager/src/main/java/com/svs/hztb/service/impl/GroupDataServiceImpl.java

```diff
@@ -33,72 +32,81 @@
 
 	@Autowired
 	GroupRepository groupRepository;
-	
+
 	@Autowired
 	UserGroupEntityRepository userGroupEntityRepository;
-	
+
 	@Autowired
 	UserRepository userRepository;
-	
+
 	@Override
 	public GroupOutput removeGroup(GroupInput groupInput) {
-		
+
 		GroupOutput groupOutput = null;
 		try {
 			groupRepository.setGroupNameFor(DEFAULT_GROUP_NAME, groupInput.getGroupId(), groupInput.getUserId());
 			groupOutput = buildGroupOutput();
-		} catch(Exception e) {
-			
+		} catch (Exception e) {
+
 		}
 		return groupOutput;
-		
+
 	}
-	
+
 	@Override
 	public GroupOutput createGroup(GroupInput groupInput) {
 		GroupOutput groupOutput = null;
 		try {
-			//Create group
+			// Create group
 			GroupEntity groupEntity = FunctionUtils.convert(groupInput, new GroupInputToEntityConverter());
 			groupRepository.saveAndFlush(groupEntity);
-		//	groupRepository.findByGroupName(groupEntity.getGroupName());
-		
-			//Add group members
+			// groupRepository.findByGroupName(groupEntity.getGroupName());
+
+			// Add group members
 			List<UserGroupEntity> userGroupList = createUserGroup(groupInput.getAddMembers(), groupEntity);
 			userGroupEntityRepository.save(userGroupList);
 			groupOutput = buildGroupOutput();
-		
-		} catch(Exception e) {
-			
+			GroupDetail groupDetail = new GroupDetail();
+			List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
+
+			groupDetail.setGroupId(groupEntity.getGroupId());
+			groupDetail.setGroupName(groupEntity.getGroupName());
+			groupDetailList.add(groupDetail);
+			groupOutput.setGroupDetailList(groupDetailList);
+
+		} catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS00108)